### PR TITLE
Fix last script in Discovery Server [10009]

### PR DESCRIPTION
**Pull request by jparisu**
Signed-off-by: jparisu <javierparis@eprosima.com>
XML configuration added to intra-process.
Scripts refactor.

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/674/)

File: docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst [hunk 1]
```diff
@@ -318,12 +317,21 @@ Compare Discovery Server with Simple Discovery
 In order to compare the ROS2 execution using *Simple Discovery* or *Discovery Service*, two scripts that
 execute a talker and many listeners and analyze the network traffic during this time are provided.
 For this experiment, ``tshark`` is required to be installed on your system.
+The configuration file is mandatory in order to avoid using intra-process mode.
+
+.. note::
+
+    These scripts require a Discovery Server closure feature that is only available from versions newer than the
+    one used in ROS 2 Foxy.
```
Comment (EduPonz) [inline docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst:325]: I'd specify the actual Fast DDS version here

**Review by EduPonz — CHANGES_REQUESTED** [review]

File: docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst (hunk 1 above)
Comment (jparisu) [reply docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst:325]: It is specified in the very next line, I think it is enough

Comment (EduPonz) [reply docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst:325]: The thing is that the version in Foxy might change in the future. Maybe it's better to remove the reference to Foxy altogether.

File: docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst [hunk 2]
```diff
@@ -321,8 +321,8 @@ The configuration file is mandatory in order to avoid using intra-process mode.
 
 .. note::
 
-    These scripts require a Discovery Server closure feature that is only available from versions newer than the
-    one used in ROS 2 Foxy.
+    These scripts require a Discovery Server closure feature that is only available from versions newer than
+    Fast DDS v2.1.0.
```
Comment (EduPonz) [inline docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst:325]:
```suggestion
    These scripts require a Discovery Server closure feature that is only available from Fast DDS v2.1.0 and forward.
```

**Review by EduPonz — CHANGES_REQUESTED** [review]

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/698/)

File: docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst (hunk 2 above)
Comment (jparisu) [reply docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst]: :+1:

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/700/)

**Review by EduPonz — APPROVED** [review]

**Pull request merged**